runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/ContactSection/ContactForm/FormFields/FormFields.tsx

```diff
@@ -45,7 +46,8 @@ export const FormCheckbox: FC<CheckboxProps> = (props) => (
     <div className={styles.checkboxWrapper}>
         <input
             className={styles.checkbox}
-            placeholder=" "
+            placeholder={props.placeholder ?? ' '}
+            disabled={props.disabled}
```

Comment:
your interface FormInputProps<T> had property disabled as optional. Imo change it there to mandatory or here to optional props?.disabled